src/overlays/actors/ovl_Bg_Menkuri_Eye/z_bg_menkuri_eye.h

```diff
@@ -8,9 +8,9 @@ struct BgMenkuriEye;
 
 typedef struct BgMenkuriEye {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x4];
-    /* 0x0150 */ Collider collider;
-    /* 0x0168 */ char unk_168[0x48];
+    /* 0x014C */ s16 unk_14C;
```

Comment:
I think `unk_14C` is something like `framesUntilDisable`, it gets set to some positive value when `D_8089C1A0` (which I assume is the amount of eyes "enabled") is incremented, ticks down, and decrements `D_8089C1A0` when it reaches 0 again